styles.css

```diff
@@ -1243,8 +1330,736 @@ footer {
   flex-grow: 0;
   margin: 17px 82px;
   list-style: none;
+  justify-content: center;
 }
 
 /* Contact Form section style end here */
 
 /* ------------------------------------------------------------------------------------------------------------------ */
+
+/* ------------------------------------------------------------------------------------------------------------------ */
+
+/* Mobile Media query section starts here */
+
+@media only screen and (max-width: 1180px) {
```

Comment:
- [ ] You should use mobile first approach to style your website and use a media query for screen size greater or equal to 768px for desktop version.
![mobile-first-media-query](https://user-images.githubusercontent.com/44057863/167269468-cf297ab3-86b9-4950-8cd3-400d801686f9.png)